### PR TITLE
Add tabbing to horizontal scroll component flippers

### PR DESCRIPTION
**Pull request by robarbms**
# Pull Request

## 📖 Description

Users should be able to use the keyboard to navigate the horizontal-scroll component.

### 🎫 Issues

[fix: should be able to tab to the flipper control in horizontal-scroll component ](https://github.com/microsoft/fast/issues/4713)

## 👩‍💻 Reviewer Notes

Converted the flipper wrappers to buttons so that they could get the click / enter key events bubble from the flippers. Disabled tab on the wrapper and enabled tabbing on the flippers.

### General

- [x] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

### Component-specific

- [ ] I have added a new component
- [x] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## ⏭ Next Steps

Add default flippers using the fast-flipper component. I'll create a separate issue.

### CODE REVIEW

**Comment by scomea** [comment]
It looks like we're relying on authors to set "aria-hidden" on the flippers they provide.  Should the scroll component force or default that setting to false if unset?

File: packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts
Comment (chrisdholt) [inline packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts:25]: If I slot a flipper we have buttons inside buttons essentially - I think we discussed potentially doing this in the feature branch to leverage the new API for default slotted content. I think that makes this story a lot easier.

File: packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts
Comment (chrisdholt) [inline packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts:57]: Same note as above - I think we should target the feature branch on this and leverage the default slotted content API's. In that case, we can also slot the actual flippers themselves.

**Review by chrisdholt — CHANGES_REQUESTED** [review]
I think if we target the feature branch we can leverage the default slotted content API's and create a really robust story here. I'm still of the mind that by *default* the flippers should be hidden from AT. If we leverage the new API for default slotted content, we can actually do some clever things to make bringing these into the tab index.

1. We can default the actual flippers, which is more reuse
2. We can expose an attribute that can be used to toggle visibility of the defaulted flippers
3. Because the scroll handlers are public methods, we can leverage those on the default slotted content - when someone slots their own, they can still leverage it.

File: packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts
Comment (robarbms) [reply packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts:25]: @chrisdholt  I wasn't sure if I should wait until vNext was merged back into master.

**Pull request closed without merging**
